packages/slice-machine/src/components/PageLayout/PageLayout.tsx

```diff
@@ -3,17 +3,21 @@ import type { FC, PropsWithChildren } from "react";
 import * as styles from "./PageLayout.css";
 
 type PageLayoutProps = PropsWithChildren<{
-  activeEnvironmentKind?: "prod" | "stage" | "dev";
+  environmentKind?: "prod" | "stage" | "dev";
```

Comment:
ℹ️ Just so you know, PageLayout should be product functional agnostic, meaning no mention to environement here. @bapmrl will open a PR I think for the future.

Reply:
Yep makes sense